Makefile

```diff
@@ -616,8 +616,21 @@ $(examples) : $(libceed)
 $(tests) : $(libceed)
 $(tests) $(examples) : override LDFLAGS += $(if $(STATIC),,-Wl,-rpath,$(abspath $(LIBDIR))) -L$(LIBDIR)
 
+# Set number processes for testing
+NPROC_TEST ?= 1
+export NPROC_TEST
+
+# Make sure the backend list is separated with spaces and not newlines
+EMPTY :=
+SPACE := $(EMPTY) $(EMPTY)
+define \n
+
+
+endef
+BACKENDS_LIST := $(subst ${\n},$(SPACE),$(BACKENDS))
```

Comment:
I don't particularly like this -- it is only necessary because the GitLab CI passes a `\n` separated list of backends instead of a space-separated one. @jedbrown  or @jeremylt, I'm not sure how to edit those pipelines (if that's possible)

Reply:
`BACKENDS_CPU=$(make info-backends-all | grep -o '/cpu[^ ]*')`

This is the magic evocation that's creating the hassle, I think

Reply:
Yep, it is

Reply:
we could definitely just pipe that into sed to replace the newlines with spaces

Reply:
```
BACKENDS_CPU=$(make info-backends-all | grep -o '/cpu[^ ]*' | sed -e 's/\n/ /g')
```
I think this would work

Reply:
The pattern is here: https://github.com/CEED/libCEED/blob/main/.gitlab-ci.yml